README_en.md

```diff
@@ -0,0 +1,69 @@
+[中文版](README.md)
+
+[![BUILD Status](https://img.shields.io/jenkins/build?jobUrl=http%3A%2F%2F59.111.93.165%3A8080%2Fjob%2Fopencurve_multijob%2F)](http://59.111.93.165:8080/job/opencurve_multijob/lastBuild)
+[![Jenkins Coverage](https://img.shields.io/jenkins/coverage/cobertura?jobUrl=http%3A%2F%2F59.111.93.165%3A8080%2Fjob%2Fcurve_untest_job%2F)](http://59.111.93.165:8080/job/curve_untest_job/HTML_20Report/)
+
+CURVE is a distributed storage system designed and developed independently by NetEase, featured with high performance, high availability, high reliability and well expansibility, and it can serve as the basis for storage systems designed for different scenario (e.g. block storage, object storage and cloud database). 
+
+So far, we have implemented a high performance block storage system, which supports snapshot, clone and recovery, and can be mounted by QEMU and NBD devices. CURVE has been served as a cloud disk inside NetEase for a certain time, during which high performance and reliability have shown.
```

Comment:
and can be mounted by QEMU and NBD devices-->and can be attached on QEMU virtual machine or physical machine(by curve-nbd)